src/app/legacy/containers/Navigation/index.jsx

```diff
@@ -22,11 +22,14 @@ const renderListItems = (
   clickTrackerHandler,
   viewRef,
 ) =>
-  navigation.map((item, index) => {
-    const { title, url } = item;
+  navigation.reduce((listAcc, item, index) => {
+    const { isLite } = useContext(RequestContext);
```

Comment:
Should we pass in `isLite` as a function argument rather than creating a new instance of `useContext(RequestContext)` on each navigation item?